h/static/scripts/controllers/autosuggest-dropdown-controller.js

```diff
@@ -0,0 +1,431 @@
+'use strict';
+
+const Controller = require('../base/controller');
+const setElementState = require('../util/dom').setElementState;
+
+let CONTAINER_CLASS = '';
```

Comment:
Since these are global variables, if two instances of the autosuggest component are created and both use different styles, whichever is created last will win and set the styles used by the other instance - at least for dynamic updates.